src/containers/CustomRuns/CustomRuns.jsx

```diff
@@ -227,7 +243,14 @@ function CustomRuns() {
       },
       {
         actionText: intl.formatMessage({
-          id: 'dashboard.cancelTaskRun.actionText',
+          id: 'dashboard.editAndRun.actionText',
+          defaultMessage: 'Edit and run'
+        }),
+        action: editAndRun
+      },
+      {
+        actionText: intl.formatMessage({
+          id: 'dashboard.cancelCustomRun.actionText',
```

Comment:
No need to change this string, we can continue using the existing one:

```suggestion
          id: 'dashboard.cancelTaskRun.actionText',
```